docs/reference/migration/migrate_8_0/_cluster-node-setting-changes.asciidoc

```diff
@@ -0,0 +1,891 @@
+[discrete]
+[[breaking_80_cluster_node_setting_changes]]
+==== Cluster and node setting changes
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+//tag::notable-breaking-changes[]
+TIP: {ess-setting-change}
+
+.You can no longer set `xpack.searchable.snapshot.shared_cache.size` on non-frozen nodes. {ess-icon}
+[%collapsible]
+====
+*Details* +
+Setting `xpack.searchable.snapshot.shared_cache.size` to be positive on a node
+that does not have the `data_frozen` role was deprecated in {es} 7.12.0 and has
+been removed in {es} 8.0.0.
+
+*Impact* +
+{es} only allocates partially mounted indices to nodes with the `data_frozen`
+role. Do not set `xpack.searchable.snapshot.shared_cache.size` on nodes without
+the `data_frozen` role. Removing the setting on nodes without the `data_frozen`
+role will not impact functionality.
+====
+
+.`action.destructive_requires_name` now defaults to `false`. {ess-icon}
+[%collapsible]
+====
+*Details* +
+The default value of the setting `action.destructive_requires_name` changes from `false`
+to `true` in {es} 8.0.0.
+
+In previous versions, the default setting allowed users to use wildcard
+patterns to delete, close, or change index blocks on indices. In order
+to prevent the accidental deletion of indices that happen to match a
+wildcard pattern, we now require, by default, that any such destructive
+operation explicitly name the indices it intends to modify.
```

Comment:
```suggestion
The default for the `action.destructive_requires_name` setting changes from `false`
to `true` in {es} 8.0.0.

Previously, defaulting to `false` allowed users to use wildcard
patterns to delete, close, or change index blocks on indices. 
To prevent the accidental deletion of indices that happen to match a
wildcard pattern, we now default to requiring that destructive
operations explicitly name the indices to be modified.
```